src/mongodb/config/language/or.ts

```
@@ -0,0 +1,6 @@
+var or_config = {
+    vowel: ["ା", "ି", "ୀ", "ୁ", "ୂ", "ୃ", "ୄ", "େ", "ୈ", "ୋ", "ୌ", "ઃ"],
```

ESLint

[error] Replace `··vowel:·["ା",·"ି",·"ୀ",·"ୁ",·"ୂ",·"ୃ",·"ୄ",·"େ",·"ୈ",·"ୋ",·"ୌ",·"ઃ"` with `vowel:·['ା',·'ି',·'ୀ',·'ୁ',·'ୂ',·'ୃ',·'ୄ',·'େ',·'ୈ',·'ୋ',·'ୌ',·'ઃ'`

(prettier/prettier)